build.config

```diff
@@ -143,7 +151,8 @@ OLA_LOGFILES = {
   "include-ola-io-test-suite.log": "include/ola/io/test-suite.log",
   "include-ola-test-suite.log": "include/ola/test-suite.log",
   "include-ola-math-test-suite.log": "include/ola/math/test-suite.log",
-  "include-ola-messaging-test-suite.log": "include/ola/messaging/test-suite.log",
+  "include-ola-messaging-test-suite.log":
+  "include/ola/messaging/test-suite.log",
```

Comment:
This doesn't seem very readable to me, but appears to be the only wake to make the flake8 tests pass. I'll also try and fix up the Perl above so it generates in this style when necessary.